cmd/githubrunnerscalesetlistener/main.go

```diff
@@ -160,6 +162,16 @@ func validateConfig(config *RunnerScaleSetListenerConfig) error {
 }
 
 func newActionsClientFromConfig(config RunnerScaleSetListenerConfig, creds *actions.ActionsAuth, options ...actions.ClientOption) (*actions.Client, error) {
+	if config.ServerRootCA != "" {
+		pool := x509.NewCertPool()
```

Comment:
should we use the `SystemCertPool` as default?

Reply:
I would centralize this in the actions client itself so we don't allow ourselves (me) to forget about it in different places.